### PR TITLE
Implemented basic UI + integrated Kaj's task list

### PR DESCRIPTION
**Pull request by jessechong**
Screenshot_20190311-133015_Expo

The groundwork for the UI has been implemented. The next step would be to actually make the buttons functional.

Here's how I imagine the UI:
I was thinking that the user could swipe through 2 cards: "Tasks" and "Events".

> 1. Tasks Card

As tasks are recommended to a user, the task card will get populated.
Users can swipe through their tasks and click on them to view a description on how to perform the task. Once the task is finished, the user can click a "done" button which will then prompt the user with a survey that will feed data into the user's personal model (their state/context/whatever we decided on).

> 2. Events Card

Upcoming events will populate this card.
Users can swipe through their events and click on them to view a description of the event (location, time).
Once the event is finished, users can click "done" and will be prompted with a survey that will feed data into the user's personal model.

> Hamburger menu
Need some suggestions for this.
Clicking this would allow a user to possibly:
View their personal profile? 
View the database of tasks? (so they can revisit old tasks)
View their journal? (Maybe the survey will also have a small journal functionality so users can revisit past journals?)
Etc.?

### CODE REVIEW

**Comment by jessechong** [comment]
btw i used "react native elements" for header (hamburger, title, home) and the card.

in retrospect, the card is something super easy to just style by hand. not sure what advantage there is to using another dependency. Maybe if we want the app to be more "lite" i can try to make a custom header and card.

File: backend/Dockerfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ RUN pip install -r requirements.txt
 
 ENTRYPOINT ["./server_setup_entrypoint"]
 
-EXPOSE 5000
+EXPOSE 80
```
Comment (kajdreef) [inline backend/Dockerfile]: Both ports 80 and 5000 need to be exposed, so could you add an additional line?
`EXPOSE 5000`

File: mesa/src/task.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { StyleSheet, Text, View, Button } from 'react-native';
+
+
+export class Task extends Component {
+    render() { // TODO make the task clickable.
+        return (
+            <View style={styles.taskitem}>
+                <Text>Task: {this.props.name}</Text>
+                <Button
+				  // onPress={onPressLearnMore}
```
Comment (kajdreef) [inline mesa/src/task.js]: uncomment onPress and create an empty function for onPressLearnMore like this:

```javascript
function onPressLearnMore() {
    console.log("Learn more...")
}
```

File: backend/Dockerfile (hunk 1 above)
Comment (jessechong) [reply backend/Dockerfile]: Done

**Comment by kajdreef** [comment]
Looks good! I am merging it now! 👍

**Pull request merged**